src/Components/AdditionalRepositoriesTable/AdditionalRepositoriesTable.js

```diff
@@ -98,9 +110,32 @@ const AdditionalRepositoriesTable = (props) => {
     />
   );
 
-  const sortedRepositories = sortRepos(repositories, activeSortIndex);
+  const filteredRepositories = repositories.filter((repository) => {
+    return !deletedRepositories.some(
+      (deletedRepository) =>
+        deletedRepository.repositoryName === repository.repositoryName
+    );
+  });
```

Comment:
This seems like a lot of overhead to filter out deleted repositories. We can just rerun the query right? It looks like you are already updating the query data so we shouldn't need this anyway.